### PR TITLE
docs: expand "getting started" section

### PR DESCRIPTION
**Pull request by lpawelcz**
This PR expands `Getting Started` section with some useful information on environment setup and building example targets.

### CODE REVIEW

**Comment by lpawelcz** [comment]
Hi @mithro!
Can I ask you to look into this PR?
It is also related to https://github.com/SymbiFlow/symbiflow-docs/issues/513 and the other PR here (https://github.com/SymbiFlow/symbiflow-arch-defs/pull/2228)

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -20,46 +20,73 @@ Clone repository
...
+    export CMAKE_FLAGS="-GNinja"
+
 Build example
 -------------
 
-Enter the appropriate test build directory, depending on your target architecture and invoke the appropriate make target.
-
-Build directories depend on the architecture.
-Because of that, depending on the chosen target, a different toolchain backend will be used.
+Various targets depend on specific architectures. Each architecture uses different toolchain backend that will be called during build.
```
Comment (mithro) [inline docs/source/getting-started.rst:40]:
```suggestion
While different architectures provide different build targets, there are some targets that should exist for all architectures. 

Each architecture has its own toolchain backend that will be called during build.
```

**Review by mithro — APPROVED** [review]
Looks like a pretty good improvement! Some minor comments. Feel free to merge after fixing them.

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -20,46 +20,73 @@ Clone repository
...
 (See `Project X-Ray <https://prjxray.readthedocs.io/en/latest/>`_
 and `Project Trellis <https://prjtrellis.readthedocs.io/en/latest/>`_ for more information)
 
-Moreover, it is worth to note that target names have the form <*testname_platform_outputformat*>.
-
-Assuming that you would like to generate bitstream ``.bit`` file with the counter example for the Arty board, which uses Xilinx Artix-7 FPGA, you will type:
+For development purposes a set of test designs was included for each supported architecture. In order to perform build
```
[the remark this answers is not in the document]
Comment (lpawelcz) [reply docs/source/getting-started.rst]: Fixed this

File: docs/source/getting-started.rst (hunk 1 above)
Comment (lpawelcz) [reply docs/source/getting-started.rst]: Included requested change

**Pull request merged**